### ISSUE TITLE
warning cannot modify header information.

### ISSUE THREAD

**Issue by mxmkmarquette** [chasecmiller/Crumbls-Cache#44]
GIF89a�����,D;
Warning: Cannot modify header information - headers already sent by (output started at /srv/users/serverpilot/apps/madmax/public/neufsvies/wp-content/plugins/crumbls_cache/plugin.php:1014) in /srv/users/serverpilot/apps/madmax/public/wp-includes/pluggable.php on line 1210

**Comment by chasecmiller** [comment]
Weird.  It looks like an image has been sent to it.  That's a header for a GIF image, from the looks of it.

**Comment by mxmkmarquette** [comment]
yeah and also it's saying when we activate the plugin that "The plugin generated 307 characters of unexpected output during activation. If you notice “headers already sent” messages, problems with syndication feeds or other issues, try deactivating or removing this plugin."
